config/routes.rb

```diff
@@ -1,8 +1,17 @@
 Rails.application.routes.draw do
 
   resources :companies do
-    resources :jobs
+    resources :contacts
+    resources :jobs do
+      resources :comments
```

Comment:
this is a little too far nested. would like to see a new resource with jobs and comments nested in jobs